tracing/trace_keyspace_helper.cc

```diff
@@ -313,7 +313,7 @@ cql3::query_options trace_keyspace_helper::make_slow_query_mutation_data(const o
         cql3::raw_value::make_value(uuid_type->decompose(session_records.session_id)),
         cql3::raw_value::make_value(timestamp_type->decompose(millis_since_epoch)),
         cql3::raw_value::make_value(timeuuid_type->decompose(start_time_id)),
-        cql3::raw_value::make_value(utf8_type->decompose(query_str_it->second)),
+        cql3::raw_value::make_value(utf8_type->decompose(std::move(query_str))),
```

Comment:
and I agree with Vlad's comment from v1, that not storing any value (a.k.a. null created with `cql3::raw_value::make_null()`) would be better than an empty string in case we don't have a `query` string to copy.